health-monitors/syslog-health-monitor/pkg/xid/xid_handler.go

```diff
@@ -146,21 +146,9 @@ func (xidHandler *XIDHandler) createHealthEventFromResponse(
 	}
 
 	if xidResp.Result.Metadata != nil {
-		if gpc, ok := xidResp.Result.Metadata["GPC"]; ok {
+		for k, v := range xidResp.Result.Metadata {
```

Comment:
What if the side car sends additional metadata? Can we be targeted in what metadata we considered as entities impacted?